_docs/03.8-server_side_data_hydration.markdown

```diff
@@ -0,0 +1,66 @@
+---
+title:  "Server Side Data Hydration"
+permalink: docs/ss_data_hydration.html
+toplevel: "Getting Started: Intermediate"
+---
+The server side rendering in Electrode consists of two tasks, the actual rendering of content to html and the initial state loading into the redux store(server side data hydration).
```

Comment:
The server side rendering consists of two steps: creating the initial redux store data and calling `ReactDOM.renderToString` with that data.